src/Libraries/RevitNodes/Elements/DividedPath.cs

```diff
@@ -224,7 +224,12 @@ public static DividedPath ByCurvesAndDivisions(Revit.Elements.Element[] elements
         #endregion
 
         #region Static constructors
-
+        /// <summary>
+        /// Creates a revit divided path on a curve given the curve and number of division
```

Comment:
Creates a Revit divided path on the given curve with specified amount of division